content/docs/pulumi-cloud/self-hosted/components/api.md

```diff
@@ -435,3 +435,16 @@ scrape_configs:
     static_configs:
       - targets: ["api.pulumi.com"]
 ```
+
+## Deployments
+
+In order to enable [Deployments](/docs/pulumi-cloud/deployments), the following must be configured:
```

Comment:
I think we should also link to how to setup a self managed agent, as that is a key piece as well

Reply:
I have that a couple lines below